src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,81 @@
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">8</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">quo adipisci enim quam ut ab</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
+type Props = {
+  todos: Todo[];
+  handleCurrentTodo: (todo: Todo) => void;
+  currentTodo: Todo | null;
+};
 
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
+export const TodoList: React.FC<Props> = ({ todos, handleCurrentTodo, currentTodo }) => {
+  const todoCompletedClass = (value: boolean) => cn(
+    { 'has-text-danger': value === false },
+    { 'has-text-success': value === true },
+  );
 
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
-            </td>
+  const eyeClass = (selectedTodo: Todo | null, filteredTodo: Todo) => cn(
+    'far',
+    { 'fa-eye': selectedTodo === null },
+    { 'fa-eye-slash': selectedTodo && selectedTodo.id === filteredTodo.id },
+  );
 
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
+  return (
+    <>
+      {todos.length === 0 ? (
```

Comment:
```suggestion
      {!todos.length ? (
```
pls see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) and fix this king of issues for all such cases in the code